js/client/modules/@arangodb/testsuites/dump.js

```diff
@@ -157,12 +171,11 @@ class DumpRestoreHelper extends tu.runInArangoshRunner {
       this.restoreConfig.setThreads(this.restoreOptions.threads);
     }
     if (this.restoreOptions.jwtSecret) {
-      let keyDir = fs.join(fs.getTempPath(), 'jwtSecrets');
-      if (!fs.exists(keyDir)) {  // needed on win32
-        fs.makeDirectory(keyDir);
+      this.otherKeyDir = fs.join(fs.getTempPath(), 'jwtSecrets');
```

Comment:
This is the same as `this.keyDir`. Is this intentional? Can we not just reuse `this.keyDir` if they're both the same?

Reply:
ok, changed.